utils/constants.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+const (
+	// LabelDeletionPriority is a label for the resources deployed in cluster ns which are waiting to be cleaned up
+	// after cluster is deleted.
+	// The value is an integer value, The larger the value, the later the order of deletion.
+	// The deletion order is :
+	// 1. delete resources without this label firstly.
+	// 2. delete resources with invalid value of this label.
+	// 3. delete resources following the priority value. For example, there are 2 manifestWorks, one value is set 100
+	// and another is set 10, the manifestWorks with 10 will be deleted before the one with 100.
+	LabelDeletionPriority string = "open-cluster-management.io/deletion-priority"
```

Comment:
seems to me it should be a field rather than label

Reply:
changed to annotation with the name cleanup-priority.